examples/resume_training.py

```diff
@@ -1,60 +1,37 @@
 #!/usr/bin/env python3
 
 import argparse
-import json
 import os
 import subprocess
-import uuid
-
-import joblib
 
 from garage.experiment import to_local_command
-from garage.logger import logger
-from garage.np.algos import BatchPolopt
-
-filename = str(uuid.uuid4())
 
 if __name__ == '__main__':
-
     parser = argparse.ArgumentParser()
-    parser.add_argument('file', type=str, help='path to the snapshot file')
     parser.add_argument(
-        '--log_dir',
+        '--snapshot_dir',
         type=str,
         default=None,
-        help='path to the new log directory')
-    # Look for params.json file
+        help='Directory of the pickle file to resume experiment from.')
+    parser.add_argument(
+        '--resume_epoch',
+        type=str,
+        default=None,
+        help='Index of epoch to restore from. '
+        'Can be "first", "last" or a number. '
+        'Not applicable when snapshot_mode="last"')
+
     args = parser.parse_args()
-    parent_dir = os.path.dirname(os.path.realpath(args.file))
-    json_file_path = os.path.join(parent_dir, 'params.json')
-    logger.log('Looking for params.json at {}...'.format(json_file_path))
+    params = dict()
+    params['resume_from_dir'] = args.snapshot_dir
+    if args.resume_epoch is not None:
+        params['resume_epoch'] = args.resume_epoch
+    command = to_local_command(
```

Comment:
would it be easier to use the experiment wrapper directly in this example? at least there would be fewer moving parts.

Reply:
or even just LocalRunner

Reply:
Using `python -m garage.experiment.experiment_wrapper --restore_from_dir ...` does the same job. But I feel that experiment_wrapper is too cumbersome. This script hides unrelated parts of experiment_wrapper and provides a lightweight access to the resuming feature.

Reply:
ok sounds fine. we can jus try to make this experience nicer in the future